src/app/credential-issuer/middleware.js

```diff
@@ -21,7 +20,6 @@ module.exports = {
     const evidenceParam = new URLSearchParams([
       ["authorization_code", req.credentialIssuer.code],
       ["credential_issuer_id", req.query.id],
-      ["redirect_uri", `${EXTERNAL_WEBSITE_HOST}/credential-issuer/callback?id=${req.query.id}`],
```

Comment:
I think the back-end still requires this value. It passes it into the /token endpoint of the CRI for validation reasons.

The Pojo class that core-back converts this request body to is expecting this var